spec/services/planning_application_search_spec.rb

```diff
@@ -5,12 +5,14 @@
 RSpec.describe PlanningApplicationSearch do
   describe "#results" do
     let!(:planning_application1) do
-      create(
-        :planning_application,
-        work_status: "proposed",
-        created_at: DateTime.new(2022, 1, 1),
-        description: "Add a chimney stack."
-      )
+      travel_to(Date.new(2022)) do
+        create(
+          :planning_application,
+          work_status: "proposed",
+          created_at: DateTime.new(2022, 1, 1),
+          description: "Add a chimney stack."
+        )
+      end
```

Comment:
Why are you using `DateTime` and `travel_to`? You can just use `Time.current.at_beginning_of_year`.

Reply:
`Time.current.at_beginning_of_year` will return 2023. I think the test wants to search a reference with 2022 i.e. `"22-00100-LDCP"`

That created_at override can be taken out though

Reply:
Or did you mean just change that created_to to `Time.current.at_beginning_of_year` ?

Reply:
But anyway I can just do 
```ruby
let!(:planning_application1) do
      travel_to(Date.new(2022)) do
        create(
          :planning_application,
          work_status: "proposed",
          description: "Add a chimney stack."
        )
     end
   end
```

and the tests pass

Reply:
Ignore my comment about returning 2023 🤦 , within the context of that travel_to it returns 2022 obviously

Reply:
Sorry, thought the `created_at` was necessary - just using `travel_to` is fine.

Reply:
Yep I thought it was necessary too - but checked and it isn't so have dropped it 👍

Reply:
FYI, `travel_to` can take a string so you can do `travel_to "2022-01-01"` if the constructor is too ugly 🙈

Reply:
ooh nice I prefer the readability of that